core/common/src/main/java/alluxio/AbstractMasterClient.java

```diff
@@ -71,6 +73,19 @@ public synchronized InetSocketAddress getAddress() {
   }
 
   private static InetSocketAddress getAddress(String zkLeaderPath) {
-    return NetworkAddressUtils.getMasterAddressFromZK(zkLeaderPath);
+    return NetworkAddressUtils.getLeaderAddressFromZK(zkLeaderPath);
```

Comment:
NIT: I feel this private static method is unnecessary since it is a one-liner anyway. Feel free to inline it to its two references. Up to you.